test/envtest/k8s.go

```diff
@@ -0,0 +1,46 @@
+package envtest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func Namespace(ctx context.Context, t *testing.T, client ctrlclient.Client) corev1.Namespace {
```

Comment:
Could we rename it to explicitly state `CreateNamespace`? I feel that in most cases if the verb is not specified, we mean "get" not "create" and I was confused about what these methods do without reading their code.